Sources/Services/AWSS3/Tests/AWSS3Tests/DeleteObjectTaggingRequestTest.swift

```diff
@@ -0,0 +1,73 @@
+// Code generated by smithy-swift-codegen. DO NOT EDIT!
+
+@testable import AWSS3
+import ClientRuntime
+import Smithy
+import SmithyTestUtil
+import XCTest
+
+
+class DeleteObjectTaggingRequestTest: HttpRequestTestBase {
+    /// S3 clients should escape special characters in Object Keys when the Object Key is used as a URI label binding.
+    func testProtocol_S3EscapeObjectKeyInUriLabel() async throws {
+        let urlPrefix = urlPrefixFromHost(host: "s3.us-west-2.amazonaws.com")
+        let hostOnly = hostOnlyFromHost(host: "s3.us-west-2.amazonaws.com")
+        let expected = buildExpectedHttpRequest(
+            method: .delete,
+            path: "/my%20key.txt",
+            headers: [
+                "Content-Type": "application/json"
```

Comment:
Does this PR need to have its codegen updated?

Reply:
oh yes it does